00_Base/json-schema-processor.js

```diff
@@ -60,7 +60,10 @@ fs.readdir(path, (error, files) => {
               `',\n  passwordString = 'passwordString` +
               enumDefinition.substring(lastLineIndex);
           }
-          globalEnumDefinitions[enumName] = enumDefinition;
+          globalEnumDefinitions[enumName] = enumDefinition.replaceAll(
+            'export const enum',
```

Comment:
There's a better way to do this. 
See: https://github.com/bcherny/json-schema-to-typescript?tab=readme-ov-file#options
We can set enableConstEnums to false in the jsts.compile call around line 140